ansible/roles/metrics/files/process-cloudflare/process-cloudflare.js

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/env node
+
+'use strict'
+
+//const { pipeline, Transform } = require('stream')
+const { Transform } = require('stream')
+const { pipeline } = require('stream').promises
+const split2 = require('split2')
+const strftime = require('strftime').timezone(0)
+const {Storage} = require('@google-cloud/storage');
```

Comment:
This file has a mix of semicolons/ASI. It would be nice if there was some consistency.